docs-source/content/faq/boot-identity-not-valid.md

```diff
@@ -0,0 +1,71 @@
+[WebLogic domain directory](https://docs.oracle.com/middleware/12213/wls/DOMCF/config_files.htm#DOMCF140). Each WebLogic
+domain directory contains a "security" subdirectory that contains a file called "SerializedSystemIni.dat".  This file contains
+security data to bootstrap the WebLogic domain, including a domain-specific encryption key.
+
+During introspection, the operator generates a Kubernetes job that runs a pod in the domain's Kubernetes namespace and with the
+same Kubernetes service account that will be used later to run the Administration Server. This pod has access to the Kubernetes
+secret referenced by `weblogicCredentialsSecret` and encrypts these values with the domain-specific encryption key so that the
+secured value can be injected as the "boot.properties" contents when starting server instances.
+
+When the domain directory is changed such that the domain-specific encryption key is different, the "boot.properties" entries
+generated during introspection will now be invalid.
+
+This can happen in a variety of ways, depending on the [model selected](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/choosing-a-model/):
+
+#### Domain in a Docker image and rolling to an image containing new or unrelated domain directory
+
+The domain is using the "domain in a Docker image" model and the error occurs while rolling to a new Docker image that contains an entirely new or unrelated domain directory.
+
+The problem is that WebLogic cannot support server instances being part of the same WebLogic domain if the server instances do 
+not all share the same domain-specific encryption key and other bootstrapping security details. Additionally, operator introspection
+currently only happens when starting servers following a total shutdown. Therefore, the "boot.properites" generated from
+introspecting the image containing the original domain directory will be invalid when used with a container started with
+the updated Docker image containing the new or unrelated domain directory.
+
+The solution is to either follow the recommended [CI/CD guidelines](https://oracle.github.io/weblogic-kubernetes-operator/userguide/cicd/) so that the original and new Docker images contain domain directories
+with consistent domain-specific encryption keys and bootstrapping security details or to [perform a total shutdown](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/domain-lifecycle/startup/#starting-and-stopping-servers) of the domain so
+that introspection reoccurs as servers are restarted.
+
+#### Domain in a Docker image after a full domain shutdown and restart
+
+The domain is using the "domain in a Docker image" model and the error occurs while starting servers after a full domain shutdown. 
+
+For this type of failure, it is even more probable that the user name and password specified in the Kubernertes secret referenced by
```

Comment:
You have two causes and one solution here all kind of jammed together.  Make it easier on the reader by making a bullet list of two bullets, each with one cause and its associated solution. Perhaps:
-------
The domain is using the "domain in a Docker image" model and the error occurs while starting servers after a full domain shutdown. For this type of failure, there are two potential causes:

* It is most probable that the user name and password specified in the Kubernertes secret referenced by `weblogicCredentialsSecret` are wrong. 

* If your development model generates new Docker images with new and unrelated domain directories and then tags those images with the same tag, then different Kubernetes worker nodes may have different images under the same tag in their local Docker repositories. The simplest solution is to set `imagePullPolicy` to `Always`; however, the better solution would be to design your development pipeline to generate new Docker image tags on every build and to never reuse an existing tag.

Reply:
Along these same lines for easier read and reference, what about having sections and sub section similar to:

* Domain In Image
  - rolling to an image
    > A
  - a full domain shutdown and restart
    > A
    > B
* Domain on a persistent volume
  - replacing the domain directory
    > A